api-gateway-routes/auth_error_response_helper.rb

```diff
@@ -0,0 +1,14 @@
+module AuthErrorResponseHelper
+  AUTHORIZATION_ERROR_KEY = "authorization_error".freeze
```

Comment:
I added [logic](https://github.com/code-dot-org/code-dot-org/pull/45202/files) a while back to parse authorizer messages when they eventually come in. Can we align with that format so we don't need to do excess work or is there a reason we can't?

Reply:
Oh good call! Totally forgot we had this already. I'll update this to match the existing format.

Reply:
Updated this and our other token status to match Javalab!